src/main/java/org/moltimate/moltimatebackend/request/ActiveSiteAlignmentRequest.java

```diff
@@ -13,8 +21,22 @@
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
-public class ActiveSiteAlignmentRequest extends AlignmentRequest {
+public class ActiveSiteAlignmentRequest {
 
-    private List<String> pdbIds;
-    private String ecNumber;
+    private List<String> pdbIds = new ArrayList<>();
+    private List<String> options = new ArrayList<>();
+    private List<String> filters = new ArrayList<>();
+    private List<MultipartFile> files = new ArrayList<>(); // TODO: Rename to customMotifs
```

Comment:
Are these TODOs for later?

Reply:
Thanks! Made the change.
The ecNumber TODO is for later

Reply:
👍